great_expectations/data_context/store/store.py

```diff
@@ -271,16 +279,15 @@ def build_store_from_config(
                 config_defaults=config_defaults,
             )
         except gx_exceptions.DataContextError as e:
-            new_store = None
             logger.critical(
                 f"Error {e} occurred while attempting to instantiate a store."
             )
-        if not new_store:
             class_name: str = store_config["class_name"]
-            module_name = store_config["module_name"]
+            module_name = store_config["module_name"]  # TODO: use .get() ???
```

Comment:
I think this should be a `get()` but I'm not sure what the default should be.
1. `module_name`
2. `None`

What's more helpful for debugging purposes with the error that will be raised below??